connectors/standalone/src/main/java/io/delta/standalone/expressions/Like.java

```diff
@@ -0,0 +1,31 @@
+package io.delta.standalone.expressions;
+
+/**
+ * Evaluates {@code expr1} like {@code expr2} for {@code new Like(expr1, expr2)}.
+ */
+public class Like extends BinaryOperator implements Predicate {
+    public Like(Expression left, Expression right) {
+        super(left, right, "like");
+    }
+
+    @Override
+    protected Object nullSafeEval(Object leftResult, Object rightResult) {
+        String str = String.valueOf(rightResult);
+        String subStr;
+        if (str.startsWith("%")) {
```

Comment:
this implementation seems insufficient to handle multiple `%` usages in the regex string. I think using regex here would be much simpler.

Reply:
@scottsand-db @vkorukanti 
LGTM ,  I want to support three matching modes of SQL: like, ilke, and regex, and also support escape characters